server/rollback.go

```diff
@@ -4,9 +4,11 @@ import (
 	"fmt"
```

Comment:
This file can be reverted now too.

Reply:
Ah I didn't read right your msg on slack lol. Done, thank you